app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -28,7 +28,30 @@
     <% end %>
     <% if article.published %>
       <span id="pageviews-<%= article.id %>" class="cta pill dashboard-pageviews-indicator" data-analytics-pageviews data-article-id="<%= article.id %>">
-        <%= article.page_views_count > 100 ? article.page_views_count : "under 100" %> views // <%= article.positive_reactions_count %> <%= "reaction".pluralize(count: article.positive_reactions_count) %> // <%= article.comments_count %> <%= "comment".pluralize(count: article.comments_count) %>
+        <span class='page-views-count'>
+          <span class="value">
+            <% if article.page_views_count > 100 %>
+               <span class="value">article.page_views_count</span>
```

Comment:
I think this is a bug - that will just print the text "article.page_views_count". I think you want `<%= article.page_Views_count %>`

Reply:
I was on DEV.to today and was like, I hope that's not my bug. 😅

Reply:
Hahaha! I was on the dashboard just before the change checking stuff. Refreshed the page and there was the ability to sort - a change I knew I wanted but didn't know it was actually already been worked on. After a bit of playing with it I realised about the view count missing and took a guess it was related.

Oh well, the site is still up at least! 😂